eng/config/PublishData.json

```diff
@@ -145,7 +145,8 @@
       ],
       "vsBranch": "main",
       "vsMajorVersion": 17,
-      "insertionTitlePrefix": "[d17.3p2]"
+      "insertionCreateDraftPR": true,
```

Comment:
What's this change? Why wasn't it true before?

Reply:
Removed : )